src/client/src/orangehrmRecruitmentPlugin/pages/ViewActionHistory.vue

```diff
@@ -104,7 +104,7 @@
         <oxd-grid :cols="3">
           <oxd-grid-item class="--span-column-2">
             <oxd-input-field
-              v-model="history.note"
+              v-model="interview.note"
```

Comment:
what if the action is not an interview? this will break for other actions.

Reply:
@SandeepaOHRM  Shall I change this?

Reply:
When scheduling the interview, surely we are not setting any note for history table and setting note for interview. So, interview scheduling history note is always null. Can we do it like this if possible

`note = this.interview ? this.interview.note : this.history.note` and set the note to v-model